content/topics/redux-with-angular/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Redux
+ready: true
+prerequisites:
+  hard: ["topics/web-frontend/intro-to-angular",
+         "topics/angular-elements"]
+  soft: []
+tags: ["BDD"]
+---
+
+### Introduction to redux and the use of it with angular for state management.
+
+[Redux state management with Angular](https://app.pluralsight.com/library/courses/angular-2-redux-manage-state/table-of-contents?aid=7010a000002BWqGAAW)
```

Comment:
we are just trialing pleauralsight, this might need to be removed later.

Reply:
Noted